src/server/game/AuctionHouse/AuctionHouseMgr.h

```diff
@@ -110,6 +110,7 @@ class AuctionHouseObject
     }
 
     typedef std::map<uint32, AuctionEntry*> AuctionEntryMap;
+    typedef std::unordered_map<ObjectGuid, time_t> PlayerGetAllThrottleMap;
```

Comment:
Agreed, I considered adding something to the AH update, it's not often run and so shouldn't be a problem.